src/redux/data.ts

```diff
@@ -1125,12 +1114,18 @@ export const addressAssetsReceived = (
   );
 
   if (removed) {
-    updatedAssets = mapValues(newAssets, asset => {
-      return {
+    updatedAssets = Object.entries(newAssets).reduce<{
+      [id: string]: {
+        asset: ZerionAsset | ZerionAssetFallback;
+        quantity: number;
+      };
+    }>((acc, [key, asset]) => {
+      acc[key] = {
         ...asset,
         quantity: 0,
       };
-    });
+      return acc;
+    }, {});
```

Comment:
Does this thunk `genericAssetsFallback` get called a lot?

This is a large function already with a lot going on, just curious if we know the ROI of a change like this. Seems like swapping out this particular usage of lodash would make a very small difference.